daemon/daemon_unix.go

```diff
@@ -102,7 +102,7 @@ func getMemoryResources(config containertypes.Resources) *specs.LinuxMemory {
 	}
 
 	if config.KernelMemory != 0 {
-		memory.Kernel = &config.KernelMemory
+		logrus.Warn("Your kernel does not support kernel memory limit capabilities or the cgroup is not mounted. Limitation discarded.")
```

Comment:
This message seems inaccurate here -- to give this warning, shouldn't it check `sysInfo.KernelMemory` first like it did previously?  Alternatively, perhaps we don't warn about kernel support at all, given we're going to ignore the container setting entirely?